File: leetcode_Q9.py
class Solution(object):
    def isPalindrome(self, x):
        if x < 0:
            return False
        t = x
        rev = 0
        while x > 0:
            
            a = x % 10
            rev = rev * 10 + a
            x = x // 10
        return t == rev

File: leetcode_Q9.c
bool isPalindrome(int x){
int t=x;
      long long rev=0;
      if(x<0)
      return false;
      while(t>0)
      {
      
      int a=t%10;
      rev=rev*10+a;
      t=t/10;
      }
      return x==rev;
}
